templates/engage/openstack-made-easy.html

```diff
@@ -0,0 +1,25 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}OpenStack made easy{% endblock %}
+
+{% block meta_description %}The change from traditional, monolithic software to multi-host microservices-based big software demands that you approach the challenge from a different perspective.{% endblock %}
+
+{% block meta_copydoc %}https://app-sjg.marketo.com/#LP5655A1LA1{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="OpenStack made easy" subtitle="With the right tools, OpenStack can be easy" image="a7916513-picto-openstack.svg" url="#the-form" cta="Download the eBook" %}
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+        <p>The phase change from traditional, monolithic software to multi-host microservices-based big software demands that you approach the challenge of deployment, integration and operations from a different perspective.</p>
+        <p>This eBook explains how we used to do things, why that is no longer an economically viable approach, and what can be done to achieve technical scalability without the large economic overhead of traditional approaches to modern software.</p>
+		<p>This eBook will give you a deeper understanding into why there is a perceived complexity to the installation and operations of OpenStack and how tools like Canonical’s OpenStack Autopilot can help you build a modern, scalable, repeatable and affordable private cloud infrastructure.</p>
+    </div>
+    <div class="col-5" id="the-form">
+    {% include "engage/shared/_en_engage_form.html" with id="1584" returnURL="https://pages.ubuntu.com/rs/066-EOV-335/images/ebook_OpenStack_Made_Easy_20160726.pdf" cta="Download the eBook" %}
```

Comment:
I think the ID should be `2547`?

Reply:
I apologise, I appear to have been confused as a result of there being multiple copies of the page on Marketo. I've pushed an update that should fix this issue.